etc/nsjail/execute.cfg

```diff
@@ -0,0 +1,137 @@
+name: "compiler explorer compilation/tool sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+max_cpus: 1
```

Comment:
we probably are better off removing this, see comment below on `cgroup_cpu_ms_per_sec`

Reply:
Yeah, this will fix the rust linker I think.